src/include/function/gds/gds_object_manager.h

```diff
@@ -4,6 +4,9 @@
 
 #include "storage/buffer_manager/memory_manager.h"
 
+using namespace std;
```

Comment:
Do not use namespace in header files, otherwise it will propagate through the include chain.

Reply:
Good to know, changed everwhere, thanks.